app/src/main/res/layout/fragment_home.xml

```diff
@@ -1,22 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
If this is just going to contain one element, let's use a FrameLayout instead of a ConstraintLayout